lib/Doctrine/DBAL/Driver/DrizzlePDOMySql/Connection.php

```diff
@@ -10,12 +10,12 @@ class Connection extends PDOConnection
     /**
      * {@inheritdoc}
      */
-    public function quote($value, $type = ParameterType::STRING)
+    public function quote($input, $type = ParameterType::STRING)
```

Comment:
I'd keep it generic as `$value` everywhere. Even if a value doesn't come from input but is defined in the code, it needs to be quoted.